src/components/selectable/selectable_list/selectable_list.tsx

```diff
@@ -38,12 +38,14 @@ import {
 import { OuiHighlight } from '../../highlight';
 import { OuiSelectableOption } from '../selectable_option';
 import AutoSizer from 'react-virtualized-auto-sizer';
+/* eslint-disable  import/named */
 import {
   FixedSizeList,
   ListProps,
   ListChildComponentProps as ReactWindowListChildComponentProps,
   areEqual,
 } from 'react-window';
+/* eslint-enable  import/named */
```

Comment:
Can we avoid these disabling comments, maybe we can add some rules to eslint config?

Reply:
The issue should be resolved by the typings for react-window, which is out of the scope of this pull request. Personally, I prefer to avoid making changes to the eslint config and to keep it as limited as possible